### PR TITLE
Fix attachment previews

### PR DESCRIPTION
**Pull request by yougotwill**
Resolves https://github.com/oxen-io/session-desktop-temp/issues/56, https://github.com/oxen-io/session-desktop-temp/issues/52

### CODE REVIEW

File: ts/session/crypto/DecryptedAttachmentsManager.ts [hunk 1]
```diff
@@ -114,7 +113,7 @@ export const getDecryptedMediaUrl = async (
 
     urlToDecryptingPromise.set(
       url,
-      new Promise(async resolve => {
+      new Promise(async (resolve, reject) => {
```
Comment (yougotwill) [inline ts/session/crypto/DecryptedAttachmentsManager.ts:116]: This was a sneaky one to catch

File: ts/components/conversation/ImageGrid.tsx [hunk 2]
```diff
@@ -90,12 +103,14 @@ export const ImageGrid = (props: Props) => {
...
           attachments={attachments.slice(0, 1)}
+          imageBroken={imageBroken}
+          highlight={highlight}
           onError={onError}
           onClickAttachment={onClickAttachment}
           renderedSize={THUMBNAIL_SIDE}
           startIndex={0}
           totalAttachmentsCount={attachments.length}
-          selected={selected}
+          messageId={messageId}
```
[the remark this answers is not in the document]
Comment (yougotwill) [reply ts/components/conversation/ImageGrid.tsx:113]: Good idea fixed!

File: ts/components/conversation/message/message-content/MessageGenericAttachment.tsx [hunk 3]
```diff
@@ -0,0 +1,75 @@
...
+
+  return (
+    <StyledGenericAttachmentContainer
+      highlight={highlight}
+      selected={selected}
+      className={'module-message__generic-attachment'}
+      onClick={onClick}
+    >
+      {pending ? (
+        <div className="module-message__generic-attachment__spinner-container">
```
[the remark this answers is not in the document]
Comment (yougotwill) [reply ts/components/conversation/message/message-content/MessageGenericAttachment.tsx:43]: Thoon 👀

**Review by Bilb — APPROVED** [review]

**Comment by Bilb** [comment]
LGTM

**Pull request merged**